website/docs/r/release.html.markdown

```diff
@@ -69,7 +80,7 @@ The following arguments are supported:
 * `devel` - (Optional) Use chart development versions, too. Equivalent to version '>0.0.0-0'. If version is set, this is ignored.
 * `version` - (Optional) Specify the exact chart version to install. If this is not specified, the latest version is installed.
 * `namespace` - (Optional) The namespace to install the release into. Defaults to `default`
-* `verify` - (Optional) Verify the package before installing it. Defaults to `false`
+* `verify` - (Optional) Verify the package before installing it. Helm uses a provenance file to verify the integrity of the chart, this must be hosted alongside the chart. For more information see the [Helm Documentation](https://helm.sh/docs/topics/provenance/). Defaults to `false`
```

Comment:
Grammar suggestion: a semi-colon links together two independent clauses that are closely related in thought.

```suggestion
* `verify` - (Optional) Verify the package before installing it. Helm uses a provenance file to verify the integrity of the chart; this must be hosted alongside the chart. For more information see the [Helm Documentation](https://helm.sh/docs/topics/provenance/). Defaults to `false`.
```